src/pages/status/index.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios';
+import { Container, CircularProgress } from '@material-ui/core';
+
+import type { NodeAddresses, ChequebookAddressResponse } from '@ethersphere/bee-js'
+
+import { beeDebugApi, beeApi } from '../../services/bee';
+
+import TroubleshootConnectionCard from '../../components/TroubleshootConnectionCard';
+import NodeSetupWorkflow from './NodeSetupWorkflow';
+import StatusCard from './StatusCard';
+import EthereumAddressCard from '../../components/EthereumAddressCard';
+
+export default function Status() {
+    const [beeRelease, setBeeRelease] = useState({ name: ''});
+    const [loadingBeeRelease, setLoadingBeeRelease] = useState(false);
+
+    const [nodeApiHealth, setNodeApiHealth] = useState('');
+    const [loadingNodeApiHealth, setLoadingNodeApiHealth] = useState(false);
+
+    const [nodeHealth, setNodeHealth] = useState({ status: '', version: ''});
+    const [loadingNodeHealth, setLoadingNodeHealth] = useState(false);
+
+    const [nodeReadiness, setNodeReadiness] = useState({ status: '', version: ''});
+    const [loadingNodeReadiness, setLoadingNodeReadiness] = useState(false);
+
+    const [nodeAddresses, setNodeAddresses] = useState<NodeAddresses>({ overlay: '', underlay: [""], ethereum: '', public_key: '', pss_public_key: ''});
+    const [loadingNodeAddresses, setLoadingNodeAddresses] = useState(false);
+
+    const [nodeTopology, setNodeTopology] = useState({ baseAddr: '', bins: [""], connected: 0, depth: 0, nnLowWatermark: 0, population: 0, timestamp: ''});
+    const [loadingNodeTopology, setLoadingNodeTopology] = useState(false);
+
+    const [chequebookAddress, setChequebookAddress] = useState<ChequebookAddressResponse>({ chequebookaddress: '' });
+    const [loadingChequebookAddress, setLoadingChequebookAddress] = useState(false);
+
+    const fetchLatestBeeRelease = async () => {
+        let beeRelease = await axios.get(`${process.env.REACT_APP_BEE_GITHUB_REPO_URL}/releases/latest`)
+        setBeeRelease(beeRelease.data)
+    }
+
+    const fetchApiHealth = () => {
+        setLoadingNodeApiHealth(true)
+        beeApi.status.health()
+        .then(res => {
+            let health = res.data;
+            setLoadingNodeApiHealth(false)
+            setNodeApiHealth(health)
+        })
+        .catch(error => {
+            console.log(error)
+            setLoadingNodeApiHealth(false)
+        })
+    }
```

Comment:
Please make these custom hooks. This is becoming a bit unreadable :).

Something along the line of:
```
// Using it
import { useApiHealth } from '../hooks'
const { health, error, isLoading } = useApiHealth()

// Definition of useApiHealth hook
import { useState } from "react";

export  const useApiHealth() {
  const [health, setHealth] = useState<...not sure... | null>(null)
  const [isLoading, setLoading] = useState<boolean>(false)
  const [error, setError] = useState<Error | null>(null)

  beeApi.status.health()
      .then(res => {
          setHealth(res.data)
      })
      .catch(error => {
          setError(error)
      })
      .finally(() => {
        setLoading(false)
      })

  return { health, isLoading, error } ;
}
```

BEWARE: this is untested code :)

Reply:
see fb0321d